<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IEEE754 Converter</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="point2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>.       </string>
    </property>
   </widget>
   <widget class="QLabel" name="point1">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>110</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>. </string>
    </property>
   </widget>
   <widget class="QLabel" name="numberInDecimalLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number in decimal form:</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimiterLabel1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimeterLabel2">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimiterLabel3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimeterLabel4">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimiterLabel5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimeterLabel6">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimiterLabel7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimiterLabel8">
    <property name="geometry">
     <rect>
      <x>546</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimiterLabel9">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimiterLabel10">
    <property name="geometry">
     <rect>
      <x>636</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimiterLabel11">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string> '</string>
    </property>
   </widget>
   <widget class="QLabel" name="delimiterLabel12">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>     '</string>
    </property>
   </widget>
   <widget class="QLabel" name="numberInBynaryLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number in bynary form:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertToBynaryPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>350</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Convert to bynary</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberInDecimalLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>244</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exponentLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00000000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="fractionLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>591</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0000000000000000000000000000000000000000000000000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="signLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
